code/game/NPC_behavior.cpp

```diff
@@ -1745,7 +1745,7 @@ qboolean NPC_BSFlee( void )
 			NPCInfo->goalEntity = foundWeap;	// Change Our Target Goal
 			NPCInfo->goalRadius	= 30.0f;		// 30 good enough?
 
-			TIMER_Set(NPC, "CheckForWeaponToPickup", Q_irand(10000, 50000));
+			TIMER_Set(NPC, "CheckForWeaponToPickup", Q_irand(1000, 5000));
```

Comment:
```suggestion
			TIMER_Set(NPC, "CheckForWeaponToPickup", Q_irand(1000, 5000) * 10);
```
This will avoid the assertion (delta > RAND_MAX) and keep the original timer value :+1: